src/components/charts/sankey/sankey-tooltip/sankey-tooltip-styles.scss

```diff
@@ -53,7 +53,7 @@
 .labelValue {
   color: $dark-gray;
   text-align: right;
-  min-width: 45px;
+  min-width: max-content;
```

Comment:
We need to be aware that this solution does not fix the UI break in all browsers (but improves previous scenario :>)
https://caniuse.com/#feat=intrinsic-width

_test in edge 17_

![image](https://user-images.githubusercontent.com/6906348/48553731-a5fbec80-e8dc-11e8-9015-5223af7eaafd.png)

Reply:
Thanks Javier, good to know, I have one more task with sankey-chart/tooltips so I will try to fix it permanently there :)